types/index.esm.d.ts

```diff
@@ -857,7 +859,7 @@ export interface Plugin<O = {}> extends ExtendedPlugin {
 	 * @param {object} options - The plugin options.
 	 * @returns {boolean} `false` to cancel the chart layout.
 	 */
-	beforeLayout?(chart: Chart, args: {}, options: O): boolean | void;
+	beforeLayout?(chart: Chart, args: Record<string, never>, options: O): boolean | void;
```

Comment:
Do we want to type these more (perhaps a later PR)? For example before layout gets called with args of `{cancelable: true}`

Reply:
Update